project/activities/views.py

```diff
@@ -7,6 +7,73 @@
 
 from .forms import ActivityModelForm
 from .models import Activity
+from .serializers import ActivitySerializer
+
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+
+@api_view(['GET', 'POST'])
+def activity_list(request):
+    """
+    Circle member(s) can create a Circle Activity via a POST request
+    Circle members and coordinators can view all Circle Activities via a GET request
+    """
+    circle_id = Activity.objects.values_list("circle_id", flat=True).first()
```

Comment:
Since Activities are logically scoped to a circle, let's pass in the `circle_id` from the URL:

- `api/v1/circles/{ circle_id }/activities`